docs/user/_sources/func.txt

```diff
@@ -88,15 +88,22 @@ The path to the acquisition order file for each scan should be specified in the
 
 Field Map-Based Distortion Correction
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Distortion correction is a method that aims to reduce distortion in EPI (fMRI) images caused by inhomogeneities in the magnetic field (which often stem from differences in tissue across tissue boundaries in the head). C-PAC has the option of including field map-based distortion correction into your pre-processing pipeline, through a collection of tools produced by FMRIB's FSL- primarily, a script which prepares field maps, and the FUGUE tool. Performing field map distortion correction requires the acquisition of a phase difference image and two magnitude images. The "best of the two" magnitude images is chosen, and a final input of one phase difference file and one magnitude file are then used by the pre-processing pipeline.
+Distortion correction is a method that aims to reduce distortion in EPI (fMRI) images caused by inhomogeneities in the magnetic field (which often stem from differences in tissue across tissue boundaries in the head). C-PAC has the option of including field map-based distortion correction into your pre-processing pipeline, and two methods, **Phase Difference (PhaseDiff)** or **Phase-Encoding Polarity (Blip-up/Blip-down)** to perform distortion correction. 
+
+Performing PhaseDiff distortion correction requires the acquisition of a phase difference image and two magnitude images. The "best of the two" magnitude images is chosen, and a final input of one phase difference file and one magnitude file are then used by the pre-processing pipeline.
+Phase-Encoding Polarity (commonly known as blip-up/blip-down) employs phase-encoding direction-specific EPI field maps to correct for distortion in the direction of the phase-encoding. It uses AFNI 3dQWarp to calculate the distortion unwarp for EPI field maps of opposite/same phase encoding direction.
 
 These files are used to generate the field map during pre-processing, and they can be provided to the C-PAC pipeline through the data configuration (participant list) file. More information on how to set this data configuration file is `available here <http://fcp-indi.github.io/docs/user/subject_list_config.html>`__.
 
 The C-PAC pipeline configuration builder provides options for configuring the Distortion Correction workflow. The field maps are generated within the distortion correction workflow, and the result is subsequently sent to the functional-to-anatomical registration step (FSL FLIRT, and with Boundary-Based Registration if selected and if tissue segmentation is run), where the distortion is "un-warped" during the transform.
 
 .. figure:: /_images/fmap_dist_corr.png
 
-#. **Perform Field Map Distortion correction - [On, Off]:** Perform field map-based distortion correction.
```

Comment:
Actually, the way it is in the code- there is only "distortion_correction" and the three options are "Off", "PhaseDiff" and "Blip" (it is only one field, not three fields of On/Off).

Reply:
Never mind- we can revisit this later- see issue:
https://github.com/FCP-INDI/C-PAC/issues/1050